pkg/minishift/openshift/version/openshift_versions.go

```diff
@@ -91,19 +92,23 @@ func PrintDownStreamVersions(output io.Writer, minSupportedVersion string) {
 }
 
 func PrintUpStreamVersions(output io.Writer, minSupportedVersion string) {
-	resp, err := getResponseBody("https://registry.hub.docker.com/v2/repositories/openshift/origin/tags/")
+	dockerRegistryUrl := "https://registry.hub.docker.com/v2/repositories/openshift/origin/tags/?page_size=1024"
```

Comment:
@LalatenduMohanty Sure, you will have to change the types into which you want to un-marshall accordingly. It will be much shorter in fact, since v1 layer and name. The former is irrelevant for us and afaict all tags are there.